ci/pipeline.yml

```diff
@@ -246,12 +234,22 @@ jobs:
 
 resources:
 
-  - name: src-staging
+  - name: src-((deploy-env))
     type: git
     icon: github
     source:
-      uri: https://github.com/18F/federalist-builder
-      branch: staging
+      uri: ((git-base-url))/((builder-repository-path))
+      branch: ((git-branch))
+
+  - name: pr-((git-branch))
+    type: pull-request
+    check_every: 1m
+    source:
+      repository: ((builder-repository-path))
+      access_token: ((gh-access-token))
+      base_branch: ((git-branch))
+      disable_forks: true
+      ignore_drafts: false
 
   - name: src-production
```

Comment:
You can remove this resource and any job that uses this resource since it's declared with the instance vars for `deploy-env` and `git-branch`